src/components/NavBar.module.css

```diff
@@ -0,0 +1,58 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  background-color: #111111;
```

Comment:
I'll do a follow-on PR to use the theme colors, but I don't want to block other work.